src/components/NcAppSidebar/NcAppSidebar.vue

```diff
@@ -733,6 +737,7 @@ export default {
 			 * @type {Event}
 			 */
 			this.$emit('figure-click', e)
+			this.$attrs.onFigureClick(e)
```

Comment:
I guess we need to add an eslint exception here @ShGKme

Reply:
Yes, here this is an exception, we need to add eslint ignore next line.

An alternative is to add a prop to control the component behavior.

Reply:
> An alternative is to add a prop to control the component behavior.

Yes, but this would mean a breaking change. Fine with me as well, I don't have a preference here.

Reply:
Fixed.